plugin/src/main/java/at/helpch/chatchat/command/SwitchChannelCommand.java

```diff
@@ -57,6 +57,6 @@ public void switchChannel(final ChatUser user, @Join @Optional @NotNull final St
             return;
         }
 
-        MessageProcessor.process(plugin, user, channel, message, false);
+        MessageProcessor.process(plugin, user, channel, user.channel(), message, false);
```

Comment:
To conform with the comment on the MessageProcessor#process, change this back to:

```java
MessageProcessor.process(plugin, user, channel, message, false);
```